index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
It might be good to drop these files into a subdirectory (sandbox?) so we can see what belongs with what. Your call, though.